### PR TITLE
[SB-332] Add Arquillian dependency

### PR DESCRIPTION
**Pull request by gytis**
https://issues.jboss.org/browse/SB-332

@metacosm I think once this is merged we should be ok to tag v1 and start using it.

### CODE REVIEW

**Comment by metacosm** [comment]
Let's wait until I tag the boosters for GA before using it ;)

**Comment by gytis** [comment]
ok

File: pom.xml [hunk 1]
```diff
@@ -89,28 +98,13 @@
       <resource>
         <directory>src/main/resources</directory>
         <filtering>true</filtering>
-        <includes>
```
Comment (metacosm) [inline pom.xml:92]: Why did you remove these? I thought that it was maybe better to list explicitly which files we're filtering, though, granted, it is a little more difficult with the new Cube capability to use maven-like properties in annotations.

Comment (gytis) [reply pom.xml:92]: Some boosters have specific files such as keycloak.json, arquillian.xml etc. I think it should be safe to filter all files for the majority of the boosters. It would be quite hard to maintain the full list of inclusions.

Comment (gytis) [reply pom.xml:92]: We can always exclude files in the boosters if we see an issue, but so far there were issues with not included patters only.

**Review by metacosm — APPROVED** [review]
